### ISSUE TITLE
Alternative way of defining documents by using lists

### ISSUE THREAD

**Issue by SchlenkR** [GDATASoftwareAG/FuncyDown#6]
I would like to mention an alternative way of describing documents by just using a list instead of piping:

```fsharp

// original way
emptyDocument
|> addH1 "FuncyDown is fancy!"
|> addParagraph "Lemonade was a popular drink, and it still is!"
|> asString


// some helpers
module Document =
  let toDoc adds = adds |> List.fold (|>) emptyDocument
  let render adds = toDoc adds |> asString

// alternative way
Document.render [
  addH1 "FuncyDown is fancy!"
  addParagraph "Lemonade was a popular drink, and it still is!"
]
```

A GIST that generated the README is [here](https://gist.github.com/ronaldschlenker/17b5ed8e16f1eb5c9c30fd857aa8a0e2).

**Comment by secana** [comment]
Thanks @ronaldschlenker for the idea, I love it. I'll add the helper functions to FuncyDown and add the example to the Readme.

**Comment by secana** [comment]
Implemented in version 1.3.0
